### PR TITLE
Eng 7889 sqlcmdtest load

### PR DESCRIPTION
**Pull request by johnTcrawford**
Tests for ENG-7889, "AADIW unit tests for sqlcmd LOAD CLASSES / REMOVE CLASSES commands", adding such tests to sqlcmdtest. Also, fixes a problem where sqlcmdtest would issue "BUILD SUCCESSFUL" (and Jenkins would be blue), even when a mismatch occurred.

### CODE REVIEW

File: tests/sqlcmd/scripts/errors/load_classes1.in [hunk 1]
```diff
@@ -0,0 +1 @@
+LOAD;
```
Comment (paulmartel) [inline tests/sqlcmd/scripts/errors/load_classes1.in]: Accidentally missing final newlines here and in load_classes2.in.
It might make sense to exercise the  "lack of newline" edge case as its own case/suite -- where we scientifically clone and mutate actual working test cases by removing their final newlines -- there HAS historically been such a bug. I would NOT want a regression in the handling of missing newlines to fail seemingly random unrelated cases like this.

File: tests/sqlcmd/baselines/errors/remove_classes3.errbaseline [hunk 2]
```diff
@@ -0,0 +1 @@
+Unexpected Ad Hoc Planning Error: java.lang.RuntimeException: Error compiling query: org.voltdb.planner.PlanningErrorException: SQL Syntax error in "REMOVE sqlcmdtest.InsertEmployee" unexpected token: REMOVE
```
Comment (paulmartel) [inline tests/sqlcmd/baselines/errors/remove_classes3.errbaseline]: As I commented offline, we should have create tickets about error messages for REMOVE or LOAD syntax errors needing to reflect that these ARE valid commands and suggest correct usage rather than treating these keywords as unexpected or invalid SQL.

**Comment by paulmartel** [comment]
End of review round. Suggest: add newlines and squash merge to master. Separately, add a task ticket for sqlcmdtest coverage for missing final newline cases.

File: tests/sqlcmd/scripts/errors/load_classes1.in (hunk 1 above)
Comment (johnTcrawford) [inline tests/sqlcmd/scripts/errors/load_classes1.in]: Fixed (added missing newlines).

File: tests/sqlcmd/baselines/errors/remove_classes3.errbaseline (hunk 2 above)
Comment (johnTcrawford) [inline tests/sqlcmd/baselines/errors/remove_classes3.errbaseline]: Done - see ENG-8744 & ENG-8745; also ENG-8747, for the suggestion (above & below) of a ticket for sqlcmdtest coverage of missing final newline cases.

**Comment by johnTcrawford** [comment]
Completed Paul's suggestions, and squash-merged on the command line; so I'm closing this.
